.github/workflows/release.yml

```diff
@@ -129,7 +131,23 @@ jobs:
         with:
           name: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
           path: azureauth-${{ github.event.inputs.version }}-${{ matrix.runtime }}
-
+  
+  # Build and sign Linux binaries on Azure DevOps and publish them to GitHub and packages.microsoft.com.
+  linux_release:
+    runs-on: ubuntu-latest
+    needs: [validate]
+    steps:
+    - name: Build, Sign and Publish Linux binaries
+      run: |
+        pip install -r ./bin/requirements.txt

```

Comment:
Do we not need to setup Python, or a particular version of Python?

Reply:
Yes we need to. I forgot to update this PR.